plugins/chat/assets/javascripts/discourse/services/chat.js

```diff
@@ -68,9 +68,7 @@ export default class Chat extends Service {
     return (
       this.currentUser.staff ||
       this.currentUser.isInAnyGroups(
-        (this.siteSettings.direct_message_enabled_groups || "11") // trust level 1 auto group
```

Comment:
No need for this default TL anymore, you cannot set this setting to "" because of validation.